app/scripts/lib/url.js

```diff
@@ -89,6 +89,18 @@ function (_) {
       return origin;
     },
 
+    /**
+     * Returns true if given "uri" has HTTP or HTTPS scheme so it is navigable
+     *
+     * @param {String} uri
+     * @returns {boolean}
+     */
+    isNavigable: function (uri) {
+      var VALID_NUMBER_OF_CHARS = 8;
```

Comment:
I'm an idiot, you use `>` below. Maybe set the value to `9` and use a `>=` below to avoid confusing people like me who didn't look close enough.